src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = sourceString
+    .split(';')
+    .filter(el => el.trim())
```

Comment:
`el` and `word` it is not a good name for variable all over the code. Try `currentStyleElement`, `currentStyKey` below. Name should describe what is inside this variable.